Cargo.toml

```diff
@@ -39,15 +39,16 @@ futures-util = { version = "0.3.26", default-features = false, features = [
 http = "0.2.6"
 imbl = "2.0.0"
 itertools = "0.12.0"
-ruma = { git = "https://github.com/ruma/ruma", rev = "4c00bd010dbdca6005bd599b52e90a0b7015d056", features = [
+ruma = { git = "https://github.com/ruma/ruma", rev = "21b644ac6ae1c7d4a4f7e98a6481a3318f2deeaa", features = [
```

Comment:
Please use the original repository since your PR has been merged.

Reply:
Agreed to use the original repo, which I've done in a commit this morning,… so there's nothing to change, right? :grin: